content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,144 @@
+
+These settings are also included in the manifest as `access` and `identityProvider` parameters.
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| Access | Controls cluster access. Private access means that nodes and pods are isolated from the internet. | Private |
+| Identity Provider | You can use Palette as the IDP or a third-party application as the IDP. | Palette |
+
+#### Internal Manifest Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| namespace | The Kubernetes namespace to install the dashboard. | `kubernetes-dashboard` |
+| ClusterRole | The ClusterRole to assign to the Kubernetes Dashboard. | `read-only` |
+| certDuration | Self-signed certificate duration in hours. | 8760h (365d) |
+| certRenewal | Self-signed certificate renewal in hours | 720h (30d) |
+| enableInsecureLogin | RBAC ensures secure login. | `false` |
+| serviceType | The service type for the dashboard. Supported values are ClusterIP, LoadBalancer, and NodePort. | `ClusterIP` |
+| skipLogin | Enables or disables the skip login option on the dashboard. | `false` |
+| enableInsecureLogin | Enables non-Secure Sockets Layer (SSL) login. Dashboard login is always restricted to HTTP(S) + localhost or HTTPS and external domain. | `false` |
+| ingress:enabled | Ingress configuration to access the ClusterIP, loadBalancer, or nodePort. | `false` |
+
+## Usage
+
+To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. The Spectro Kubernetes Dashboard pack has default settings for **Access** and **Identity Provider**. These settings automatically install the latest version of the Spectro Proxy pack when the cluster is created.
+
+Changing these default settings may require some additional configuration.
+<br /> 
+
+### Access
+
+The default setting is **Private**. If you change the setting to **Public** and your cluster is in a public cloud, there is no additional configuration.  
+
+However, if you change **Access** to **Public** and your cluster is in a private cloud, you have to manually add and configure the Spectro Proxy pack to your cluster profile. For more information, refer to the [Spectro Proxy](https://docs.spectrocloud.com/integrations/frp) guide.
+<br /> 
+
+### Identity Provider 
+
+The default setting is **Palette**. If you change this setting to **Inherit from Tenant**, you must configure OpenID Connect (OIDC) authentication. You can configure OIDC two ways:
+
+- In Tenant Admin scope, navigate to **Tenant Settings > SSO**, choose **OIDC**, and provide your third-party Identity Provider details.  
+
+- In **Tenant Settings > SSO**, if you choose **None** or **SAML**, you must configure OIDC manually in the Kubernetes pack. 
+
+### Enable OIDC Authentication
+
+To enable OIDC authentication manually, copy the configuration shown in the example to the Kubernetes Pack and provide your third-party Identity Provider details.
+<br />
+
+```
```

Comment:
This configuration is only applicable to EKS. Ideally, we should showcase the same for Azure and GCP. This is where Ravi suggested we ask the Q/A team. Arvind and Resma are two names that come to mind.

Reply:
@ravi-k8 @shantanubansal Do either of you have the OIDC configuration for Azure and GCP to include in the doc?

Reply:
@ritawatson since we are providing a smooth and enhanced experience to user, as a part of that configuration is not directly visible to the user. But still In documentation we can give some configuration on How to Enable OIDC through our pack for different clouds.

@ritawatson @karl-cardenas-coding  maybe with the help of content team and testing team configuration also can be summarized in documentation for all the cloud types we support.

Reply:
@shantanubansal I need your help/information about how users can enable OIDC through the Spectro K8s Dashboard for different clouds. Can you please provide some details?